package.json

```diff
@@ -14,7 +14,11 @@
     "preview": "rsbuild preview"
   },
   "browserslist": {
-    "production": [">0.2%", "not dead", "not op_mini all"],
```

Comment:
First of all - THANK YOU, this has been bugging me for some time now.

----
Problem: opening `package.json` and saving the file (with no changes) still formats this with Biome :confused: It seems the reason for this is that the Biome extension doesn't abide by the *biome.json* rules; switching the default VS Code formatter to "JSON language features" solves the problem. 
Are you encountering the same problem?

Reply:
> Problem: opening package.json and saving the file (with no changes) still formats this with Biome 😕

🤔this doesn't occur on my side

Reply:
@misicnenad This doesn't happen on my side, so I can only assume it's something at play in your VS Code configs.

Reply:
Interesting :thinking: Will dig in a bit when I feel bored.